docs/rfcs/0002-storing-course-data.md

```diff
@@ -0,0 +1,134 @@
+## Storing course data
+
+### Abstract
+
+The aim of this RFC is to make decisions about the following:
+- The model definitions* that will be needed to store course data.
+- The plan for loading legacy course data, and keeping that data current
+  after we switch to using our own Open edX instance.
+
```

Comment:
Based on the discussion so far, I suggest we have 2 different RFCs for each of these.  I sense there is some misunderstanding about the "legacy courses" so we can deal with that a bit later .

Reply:
We're pretty deep into the discussion on the model structure and the process for loading legacy data. It would be a bit disruptive and add more overhead to split this into two. Even if it's noisy I think it's better to keep the discussion here